label_studio/annotation_templates/generative-ai/supervised-llm/config.yml

```diff
@@ -0,0 +1,83 @@
+title: "Supervised Language Model Fine-tuning"
+type: community
+group: Generative AI
+order: 1
+image: /static/templates/generative-supervised-llm.png
+details: Collecting labeled text data for training an initial LLM, focusing on task-specific performance improvement. This stage involves gathering instructions and responses to adapt the base model to a wide range of tasks, enhancing its ability to generate more accurate and useful responses.
+config: |
+  <View>
+    <Style>
+    
+    * {
```

Comment:
We should not use (and maybe even allow) such global styles, they all should be namespaced by topmost view classname. This applies for all configs in this PR.